README.md

```diff
@@ -1 +1,126 @@
-# PADIVAR-Hardware
+# PADIVAR
+
+**PADIVAR** is an open-source OBD2 vehicle diagnostic software compatible with multiple adapters. 
+It is hosted on a server connected to one (or multiple) cars, via one (or multiple) OBD2 devices, and can be accessed simultaneously by multiple clients remotely thanks to a simple TCP client.
+
+Developed by [ESIEA](https://www.esiea.fr/) students for [Le Mans Métropole](https://www.lemansmetropole.fr/).
+
+
+## Summary
+
+<!--ts-->
+   * [PADIVAR](#padivar)
+   * [Summary](#summary)
+   * [Installation](#installation)
+      * [Build](#build)
+   * [Usage](#utilisation)
+	   * [Setup](#setup) 
+		   * [Compatible OBD2 devices](#compatible-obd2-devices)
+		   * [Available commands](#available-commands)
+	   * [Connection](#connection)
+   * [License](#license)
+<!--te-->
+
+
+# Installation
+
+## Build
+
+* Download `g++` and `make`
+* Download `libbluetooth-dev` (Debian) or `libbluez3` (Mageia)
+* Clone project `git clone https://github.com/poloz-lab/PADIVAR-Hardware`
+* Compile with `make`
+
+# Usage
+
+## Setup
+
+In order to work, PADIVAR is separated in two parts:
+* a server (referred here as '*PADIVAR server*') , running either Debian or Mageia (other distributions untested)
+* a TCP client (any simple TCP client supporting CRLF works)
+
+```
+PADIVAR Architecture example:
+
+┌──────────────┐                                       ┌──────────────┐      ┌───────┐
+│ TCP Client 1 |___                                 ___| OBD Device 1 |______| Car 1 |            
+└──────────────┘   \                               /   | (Bluetooth)  |      └───────┘
+                    \                             /    └──────────────┘                 
+┌──────────────┐     \           ┌─────────┐     /     ┌──────────────┐      ┌───────┐
+│ TCP Client 2 |______[...]______| PADIVAR |____/______| OBD Device 2 |______| Car 2 |      
+└──────────────┘                 | Server  |           |   (Wi-Fi)    |      └───────┘
+                                 └─────────┘           └──────────────┘
+└───────────────┘   └───────┘   └───────────────────────────────────────────────────────┘
+Remote client(s)    Internet      PADIVAR server connected to car(s) via OBD2 device(s)
+
+```
+#### Compatible OBD2 devices
+
+PADIVAR can be used with multiple OBD2 adapters, each using different protocols.
+At this time, the following adapters are available:
+* `elm327`
+	 * `bluetooth`
+	 * `wifi`
+	 * `usb`
+* `no device` (for device independent commands)
+
+#### Available commands
+
+At this time, the following commands are available:
+* `quit` - stops the connection
+* `listPID` - list all supported PIDs
+* `description` - displays description of PID
+	* `<PID>` (hex value)
+* `units` - displays the unit of PID value
+	* `<PID>` (hex value)
+
```

Comment:
is it possible to add the command sendPID (only if a device is connected) that asks for a pid and display the value sent by the car transformed by PADIVAR?

Reply:
And repeatPID (only if a device is connected) that asks for a PID and then a time in seconds, and display the value for the time